### PR TITLE
Add extra disks and switch dt-pidone to lvms

### PR DESCRIPTION
**Pull request by lmiccini**

### CODE REVIEW

**Comment by fultonj** [comment]
+2

File: scenarios/reproducers/va-pidone.yml [hunk 1]
```diff
@@ -139,3 +143,18 @@ cifmw_test_operator_tolerations:
     effect: "NoExecute"
 cifmw_test_operator_node_selector:
   kubernetes.io/hostname: worker-3
+
+# Use EDPM image for computes
+cifmw_update_containers_edpm_image_url: "{{ cifmw_update_containers_registry }}/{{ cifmw_update_containers_org }}/edpm-hardened-uefi:{{ cifmw_update_containers_tag }}"
```
Comment (pablintino) [inline scenarios/reproducers/va-pidone.yml:148]: Are you sure you need this one that is already declared in the release files in DS?

Comment (lmiccini) [reply scenarios/reproducers/va-pidone.yml:148]: I just thought it would be nice to have it in case we run outside of zuul (and using upstream content).

**Comment by lewisdenny** [comment]
Hey @lmiccini 

Does this PR supersede this one? #2190

**Comment by lmiccini** [comment]
> Hey @lmiccini
> 
> Does this PR supersede this one? #2190

I completely forgot about that other PR.. will close that one and we'll continue here

**Review by fultonj — COMMENTED** [review]
+2 from me

**Review by lewisdenny — APPROVED** [review]
/lgtm

The code is fine to me, with the other +2 on this PR I'm happy to add my +1

**Review by fultonj — APPROVED** [review]
/approve
/lgtm

**Pull request merged**